maven-compat/src/main/java/org/apache/maven/repository/MetadataGraphNode.java

```diff
@@ -57,7 +57,7 @@ public MetadataGraphNode addIncident( MetadataGraphNode node )
         return this;
     }
 
-    public MetadataGraphNode addExident( MetadataGraphNode node )
+    public MetadataGraphNode addExcident( MetadataGraphNode node )
```

Comment:
This would be an api change. I'm not even sure if this is a correct word.
I'm fairly confident that you are using two spellings for the same concept.